ccpp/data/GFS_typedefs.meta

```diff
@@ -5672,6 +5674,12 @@
   units = index
   dimensions = ()
   type = integer
+[ntcoarsepm]
+  standard_name = index_for_coarse_pm_in_tracer_concentration_array
```

Comment:
I assume `pm` is particulate matter? Should this be spelled out?

Reply:
Yes, it is the particulate matter. I will update to spell it out.